src/ucs/arch/aarch64/cpu.h

```diff
@@ -40,4 +40,9 @@ static inline ucs_cpu_model_t ucs_arch_get_cpu_model()
     return UCS_CPU_MODEL_UNKNOWN;
 }
 
+static inline int ucs_arch_get_cpu_flag()
```

Comment:
i think that if you cannot detect architectural support for flag X, would be better to return 0, to indicated that "we are not sure that this is supported", rather than returning "UNKNOWN" which will make you skip the check.